doc/NEWS/NEWS-3.2.0.md

```diff
@@ -651,6 +651,18 @@ The following APIs are updated.
     Extension libraries which use this interface and built for older
     versions need to rebuild with adding `init_int32` function.
 
+* `rb_postponed_job` changes
```

Comment:
Is it intentional? This file is for Ruby 3.2.

Reply:
🤦 Nope, this is a mistake - thank you for catching it, I will push up a fix now.